coursefinder/static/js/postcode-picker.js

```diff
@@ -0,0 +1,40 @@
+(function ($) {
+
+    var PostcodePicker = function(wrapper) {
+        this.wrapper = $(wrapper);
+        this.setup();
+    }
+
+    PostcodePicker.prototype = {
+        setup: function() {
+            this.btn = this.wrapper.find('.course-finder-content__next-button');
+            this.form = this.wrapper.find('.course-finder-content__question-form');
+
+            this.startWatchers();
+        },
+
+        startWatchers: function() {
+            var that = this;
+
+            this.btn.click(function() {
+                that.form.submit();
+            });
+
+            this.form.submit(function() {
+                var postcode = this.postcode.value.replace(' ', '');
+                var distance = this.distance.value;
+                var queryValue = [postcode,  distance].join(',');
```

Comment:
So if the user does not select postcode can we not set queryValue?

And if the user does not select distance, can we set it to a default value of 10?